ground/uavobjgenerator/generators/gcs/uavobjectgeneratorgcs.cpp

```diff
@@ -42,8 +42,9 @@ bool UAVObjectGeneratorGCS::generate(UAVObjectParser* parser,QString templatepat
     gcsCodeTemplate = readFile( gcsCodePath.absoluteFilePath("uavobjecttemplate.cpp") );
     gcsIncludeTemplate = readFile( gcsCodePath.absoluteFilePath("uavobjecttemplate.h") );
     QString gcsInitTemplate = readFile( gcsCodePath.absoluteFilePath("uavobjectsinittemplate.cpp") );
+    QString gcsVerTemplate = readFile( gcsCodePath.absoluteFilePath("uavogcsvertemplate.h") );
```

Comment:
I don't mean to be a stickler, but it would be much easier to read if the full word "version" were used throughout. I worry that ```uavogcsvertemplate``` is hard enough to read that it will give people pause as they try to digest it. Whereas, on the flight side we call if ```flightVersionTemplate```.